public/static/app/perf-bb/ChangePointsService.js

```diff
@@ -59,6 +59,18 @@ mciModule.factory('ChangePointsService', function(
     )
   }
 
+  // Get unprocessed change points for query, return promise
+  // :param query: The mongo query to filter on.
+  function getUnprocessedChangePoints(query){
+    return Stitch.use(STITCH_CONFIG.PERF).query(function(db) {
+      return db
+        .db(STITCH_CONFIG.PERF.DB_PERF)
+        .collection(STITCH_CONFIG.PERF.COLL_UNPROCESSED_POINTS)
+        .find(query)
+        .execute()
```

Comment:
oneline? semicolons?

Reply:
In terms of semicolons, I would lean towards adding them but I was just following the conventions in this file (and it passed lint).

Personally I think multiline is probably a bit easier to read / edit.